src/main/java/com/cocodan/triplan/schedule/dto/response/DailyScheduleSpotResponse.java

```diff
@@ -0,0 +1,42 @@
+package com.cocodan.triplan.schedule.dto.response;
+
+import com.cocodan.triplan.schedule.domain.DailyScheduleSpot;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+@Getter
+@AllArgsConstructor
+@Builder
+public class DailyScheduleSpotResponse {
+
+    private Long spotId;
+
+    private LocalDate date;
+
+    private Integer order;
```

Comment:
```suggestion
    private int order;
```
primitive로 바꾸면 좋을 것 같아용~~

Reply:
반영하였습니다~